supervisor/config/boot.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+#==============================================================================
+# Copyright (C) 2021-present Alces Flight Ltd.
+#
+# This file is part of Flight File Manager.
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which is available at
+# <https://www.eclipse.org/legal/epl-2.0>, or alternative license
+# terms made available by Alces Flight Ltd - please direct inquiries
+# about licensing to licensing@alces-flight.com.
+#
+# Flight File Manager is distributed in the hope that it will be useful, but
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, EITHER EXPRESS OR
+# IMPLIED INCLUDING, WITHOUT LIMITATION, ANY WARRANTIES OR CONDITIONS
+# OF TITLE, NON-INFRINGEMENT, MERCHANTABILITY OR FITNESS FOR A
+# PARTICULAR PURPOSE. See the Eclipse Public License 2.0 for more
+# details.
+#
+# You should have received a copy of the Eclipse Public License 2.0
+# along with Flight File Manager. If not, see:
+#
+#  https://opensource.org/licenses/EPL-2.0
+#
+# For more information on Flight File Manager, please visit:
+# https://github.com/openflighthpc/flight-file-manager
+#===============================================================================
+
+ENV['RACK_ENV'] ||= 'development'
+ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
+
+require 'rubygems'
+require 'bundler'
+require 'yaml'
+require 'json'
+require 'pathname'
+
+if ENV['RACK_ENV'] == 'development'
+  Bundler.require(:default, :development)
+else
+  Bundler.require(:default)
+end
+
+# Shared activesupport libraries
+require 'active_support/core_ext/hash/keys'
+
+lib = File.expand_path('../lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
+require 'flight_file_manager'
+
+# Ensures the config has been loaded
+# XXX: Consider changing to a load method
+FlightFileManager.app.config
```

Comment:
FYI, scheduler controller has `alias_method :load_configuration, :config` to handle this.  It does read a bit better.

Reply:
0846e773e5be0e3fa6b301044db75e394098d902